engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/activity/ActivityTest.java

```diff
@@ -60,6 +60,18 @@
           .moveToActivity("task")
           .endEvent("taskEnd")
           .done();
+  private static final BpmnModelInstance WITH_STATIC_INPUT_MAPPING =
+      Bpmn.createExecutableProcess(PROCESS_ID)
+          .startEvent()
+          .serviceTask(
+              "task",
+              b ->
+                  b.zeebeJobType("type")
+                      .zeebeInput("\"text\"", "text")
+                      .zeebeInput("1", "number")
+                      .zeebeInput("true", "boolean"))
```

Comment:
❌ I think this shows why static values are not supported so far.

All other places allow a static value to be defined as a string without the `"` double quotes around it. However, here you need the quotes. When you omit the quotes, the static value will be considered the name of a variable.

Reply:
Note that this differs from `jobType`, `assignee`, `candidateGroups`, `error-name` and a few others.

Reply:
I see your point, let me look into it

Reply:
💡 You might have a look at 'escaping' the result of the source expression if it is an instanceof StaticExpression.

Reply:
The idea of the initial issue (https://github.com/camunda-cloud/zeebe/issues/8733) was to auto-wrap all non-expression inputs (`= 12`) as strings (`12` becomes `= "12"`).

Interesting case is escaping `"` inside the string, i.e. handling `My name is "walt"`.

Reply:
I agree that we should handle the static values as strings :+1: 

Static values don't have type information. We need to treat them as strings.   

Regarding the escaping of `"` in a static value, is an interesting edge case. We should create a test case for it.